File: Bars.GkhGJI/Entities/Dict/ArticleTsj.cs
using Bars.Gkh.Entities;

namespace Bars.GkhGji.Entities
{
    /// <summary>
    /// Статья ТСЖ
    /// </summary>
    public class ArticleTsj : BaseGkhEntity
    {
        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Группа
        /// </summary>
        public virtual string Group { get; set; }

        /// <summary>
        /// Статья ЖК
        /// </summary>
        public virtual string Article { get; set; }
    }
}

File: Bars.GkhGJI/libs/B4/model/Dict/ArticleTsj.js
Ext.define('B4.model.dict.ArticleTsj', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ArticleTsj'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'Code' },
        { name: 'Group' },
        { name: 'Article' }
    ]
});